pyproject.toml

```
@@ -55,7 +55,7 @@ dependencies = [
 cli = [
     "httpx-auth ~=0.23.1",
     "pyyaml ~=6.0",
-    "typer ~=0.15.4",
+    "typer ~=0.16.0",
```

```suggestion
    "typer ~=0.15.4",
```
Implement this update later